pkgs/development/python-modules/drf-yasg/default.nix

```
@@ -0,0 +1,37 @@
+{
+  stdenv,
+  buildPythonPackage,
+  fetchPypi,
+  inflection,
+  ruamel_yaml,
+  setuptools_scm,
+  six,
+  coreapi,
+  djangorestframework,
+}:
+
+buildPythonPackage rec {
+  pname = "drf-yasg";
+  version = "1.16.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0ri5h5xsacm99c6gvb4ldwisbqgiv2vq8qbn7vrh6vplzlpyvzb8";
+  };
+
+  propagatedBuildInputs = [
+    six
+    inflection
+    ruamel_yaml
+    setuptools_scm
```

this should belong in nativeBuildInputs
```suggestion
    setuptools_scm
```

---

:+1: